doc/rst/source/project.rst

```diff
@@ -147,11 +147,13 @@ Optional Arguments
 
 .. _-G:
 
-**-G**\ *dist*\ [/*colat*][**+h**]
+**-G**\ *dist*\ [/*colat*][**+c**\|\ h**]
     Generate mode. No input is read. Create (*r*, *s*, *p*) output
     points every *dist* units of *p*. See **-Q** option. Alternatively,
     append **/**\ *colat* for a small circle instead [Default is a
-    colatitude of 90, i.e., a great circle]. Use **-C** and **-E** to
+    colatitude of 90, i.e., a great circle]. If setting a pole with **-T**
+    and you want to small circle to go through *cx*/*cy*, append **+c** to
```

Comment:
"you want to small circle to go through" doesn't read well.

Reply:
ops, fixed.